src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -103,10 +103,7 @@ define(function (require, exports, module) {
                     );
 
                     var className = classnames({
-                        "layer-group": layerGroup,
-                        "layer-group__selected": layerGroup && layer.selected,
-                        "layer-group__collapsed": layerGroup && !layer.expanded,
-                        "layer-group__not-visible": layerGroup && !layer.visible
```

Comment:
As a result, I have to switch to sibling selector `.layer__not-invisible + .layer-group`, and the performance is not as good as child selector `layer-group__not-invisible > .layer-group`. I am observing 10ms - 15ms increased in rendering time.